### PR TITLE
added entity and media entity parsing

### PR DESCRIPTION
**Pull request by binaryaaron**
Referencing #16 , this PR adds media and entity parsing. I simplified the other methods in the tweet_entities.py module with the added functionality, a base method called `get_entities`.

### CODE REVIEW

**Comment by fionapigott** [comment]
Also: is there a reason you chose "https" links here? Is there ever a situation where and "http" link is available, but "https" isn't?

File: tweet_parser/getter_methods/tweet_entities.py [hunk 1]
```diff
@@ -1,6 +1,154 @@
...
+        ...          'thumb': {'h': 150, 'resize': 'crop', 'w': 150}},
+        ...          'type': 'photo',
+        ...          'url': 'https://t.co/something_else'}]}
+        ...         }
+        >>> get_media_urls(tweet)
+        ['https://pbs.twimg.com/media/something.jpg', 'https://pbs.twimg.com/media/something_else.jpg']
+    """
+
+    media = get_media_entities(tweet)
+    urls = [m.get("media_url_https") for m in media] if media else None
```
Comment (jrmontag) [inline tweet_parser/getter_methods/tweet_entities.py]: In this module, it seems like there are some inconsistencies with respect to return types e.g. here we may return `None` for `get_media_urls()` but it looks like `get_hashtags()` will return an empty list. Am I reading that correctly? If not, it'd be nice to choose a convention and make sure this module (and all other getters) follow the same pattern.

Comment (fionapigott) [reply tweet_parser/getter_methods/tweet_entities.py]: That's a good point, I think that an empty list is better here than "none"

Comment (binaryaaron) [reply tweet_parser/getter_methods/tweet_entities.py]: I'll make this change; thanks.

**Comment by binaryaaron** [comment]
@fionapigott - added the methods to the Tweet object and managed other changes as requested.

**Pull request merged**